pages/ksphere/konvoy/latest/release-notes/index.md

```diff
@@ -15,6 +15,48 @@ enterprise: false
 
 <p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. For new customers, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download Konvoy.</p>
 
+### Version v1.4.2 - Released 24 March 2020
+
+#### Disclaimer
+
+-   The default value of `vpc.enableVPCEndpoints` was changed to `false` to prevent Konvoy unexpectedly modifying the endpoints in user provided VPCs.
+    This value should already be present in your `cluster.yaml` file, below is a partial `cluster.yaml` that contains the value you may add to retain the previous behavior of deploying VPC endpoints in your cluster.
+    This resource should only be required where the networking configuration of the cluster does not allow for direct access to the AWS API.
+
+    ```yaml
+    kind: ClusterProvisioner
+    apiVersion: konvoy.mesosphere.io/v1beta1
+    metadata:
+      name: konvoy
+    spec:
+      provider: aws
+      aws:
+        vpc:
+          enableVPCEndpoints: true
+    ```
+
+#### Improvements
+
+- Update default Kubernetes version to `v1.16.8` which fixes a [CVE](https://github.com/kubernetes/kubernetes/issues/89378) found in a previous version.
+- Update Calico to `v3.13.1` which fixes a [CVE](https://www.projectcalico.org/security-bulletins/#TTA-2019-003) found in a previous version.
+- Update Ansible to `2.7.16.0` which fixes a CVE found in a previous version.
+- Improve error output when addons can not be retrieved.
+- Change the default value of `vpc.enableVPCEndpoints` to `false` to prevent Konvoy unexpectedly modifying the endpoints in user provided VPCs.
+- Provide a new option `vpc.overrideDefaultRouteTable` to disabling Konvoy modifying the route table.
+
+#### Bug fixes
+
+- Fix a bug where provisioning would fail looking up the default AWS AMI in certain regions, even if `imageID` is set.
+- In certain environments, the kubelet service can be in a started state after the package is installed, always stop that service to avoid `kubeadm` preflight errors.
```

Comment:
```suggestion
- In certain environments, the kubelet service can be in a started state, after the package is installed. Always stop that service to avoid `kubeadm` preflight errors.
```